app/src/main/java/com/google/droidjump/achievements_data/AchievementsAdapter.java

```diff
@@ -85,30 +88,34 @@ private void fillViewItems() {
         return new AchievementViewHolder(view);
     }
 
+    @SuppressLint({"UseCompatLoadingForDrawables", "NewApi"})
     @Override
-    public void onBindViewHolder(@NonNull RecyclerView.ViewHolder mholder, int position) {
-        if (items.get(position).getType() == ItemType.SECTION_NAME) {
-            SectionViewHolder holder = (SectionViewHolder) mholder;
-            holder.getSectionText().setText(sectionNames.get(items.get(position).getListPosition()));
+    public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int position) {
+        if (holder instanceof SectionViewHolder) {
+            ((SectionViewHolder) holder).getSectionText().setText(sectionNames.get(items.get(position).getListPosition()));
```

Comment:
nit: I would actually move some part of this, for example
```
sectionNames.get(items.get(position).getListPosition()
```
into separate variable to improve readability.